tests/python/frontend/onnx/test_forward.py

```diff
@@ -85,6 +85,13 @@ def get_caffe2_output(model, x, dtype='float32'):
     return c2_out
 
 
+def get_onnxruntime_output(model, x, dtype='float32'):
+    import onnxruntime.backend
+    rep = onnxruntime.backend.prepare(model, 'CPU')
+    ort_out = rep.run(x)[0]
```

Comment:
We should add `x.astype(dtype)` here.

Reply:
Thanks for the review and the issue had been updated.